openedx/core/djangoapps/api_admin/tests/test_forms.py

```diff
@@ -34,9 +34,13 @@ def test_render_value(self):
         """
         dummy_string_value = 'staff, verified'
         input_field_name = 'viewers'
-        expected_widget_html = u'<input type="text" name="{input_field_name}" value="{serialized_value}" />'.format(
+        extra_formating = ''
+        if django.VERSION < (2, 0):
```

Comment:
The markup changed in 2.1, not 2.0

Reply:
yep, you are right, I copy pastayed wrong.